modules/private-clusters-setting-dns-private.adoc

```diff
@@ -6,11 +6,17 @@
 [id="private-clusters-setting-dns-private_{context}"]
 = Setting DNS to private
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+After you deploy a cluster, you can modify its DNS to use only a private zone, by modifying the `DNS` custom resource (CR).
```

Comment:
nit: I would keep the spelled-out "custom resource" in later places, as not everyone reads the intro.

Reply:
This is specific to the docs guidelines for OpenShift and is common practice in our modules